api/Cargo.toml

```diff
@@ -14,12 +14,13 @@ actix-web = "2.0"
 actix-web-actors = "2.0"
 actix-rt = "1.0"
 base64 = "^0.12.0"
+chrono = "0.4"
 config = { version = "0.10", default-features = false, features = ["json"] }
-diesel = { version = "^1.4.3", features = ["postgres", "r2d2", "uuidv07"] }
+diesel = { version = "^1.4.3", default-features = false, features = ["chrono", "postgres", "r2d2", "uuidv07"] }
 env_logger = "0.7"
 failure = "0.1"
 gdlk = { path = "../core" }
-juniper = "0.14.2"
+juniper = { version = "0.14.2", default-features = false, features = ["chrono"] }
```

Comment:
Rust's stdlib doesn't have date support, `chrono` is the most popular lib for it